h2o-docs/src/product/explainability.rst

```diff
@@ -0,0 +1,213 @@
+Model Explainability
+====================
+
+H2O offers a convenient interface to many explainabilty methods in H2O.  The functions work for individual models, as well a list of models or an `H2O AutoML object <automl.html>`__
+
+
+Explainability Interface
+------------------------
+
+The H2O explainability interface is designed to be automatic -- all of the explainability feedback is generated with a single function, ``h2o.explain()``, which takes an object (one of the following: an H2O model, a list of models, an H2OAutoML object, an H2OAutoML Leaderboard slice), and a test frame.
+
+
+.. tabs::
+   .. code-tab:: r R
+
+        # Explain a model
+        exm <- h2o.explain(model, test)
+        print(exm)
+
+        # Explain an AutoML object
+        exa <- h2o.explain(aml, test)
+        print(exa)
+
+
+   .. code-tab:: python
+
+        # Explain a model
+        exm = model.explain(test)
+        print(exm)
+
+        # Explain an AutoML object
+        exa = aml.explain(test)
+        print(exa)
+
+
+
+
+Parameters
+~~~~~~~~~~
+
+TO DO: This was created looking at the R interface, may need some updates for Python.  
+
+- **object**: (R only) One of the following: an H2O model, a list of models, an H2OAutoML object, or an H2OAutoML leaderboard slice.
+
+- **test_frame**: An H2OFrame of test data.
+
+- **columns_of_interest**: A vector of column names. If specified, create plots only with these columns (where applicable).
```

Comment:
together with `top_n_features` changed to `columns`:
```
#' @param columns A vector of column names to create plots only with or a number of columns to pick
#'                 using variable importance (where applicable).
```

Reply:
TO DO: We are going to change this back to two paramters and I will need to update this